### PR TITLE
Adapts the default page layout for blog and blog posts

### PR DESCRIPTION
**Pull request by froschdesign**
|    Q          |   A
|-------------- | ------
| Enhancement | yes

### CODE REVIEW

File: src/Blog/templates/list.phtml [hunk 1]
```diff
@@ -14,7 +14,12 @@ $this->end();
 <section class="blog container">
     <div class="row align-items-start mt-5">
         <div class="col">
-            <h1><?= $this->escapeHtml($title) ?></h1>
+            <h1>
+                Blog
+                <?php if (isset($tag)) : ?>
+                    <small>Tag: <?= $this->escapeHtml($tag) ?></small>
```
Comment (michalbundyra) [inline src/Blog/templates/list.phtml]: can we have just one tag?

Comment (froschdesign) [reply src/Blog/templates/list.phtml]: This is only for listing blog posts via a tag. A single blog post can have multiple tags.

https://github.com/laminas/getlaminas.org/blob/3c490f76e05ec9f2e9653f1d16ac117defa376aa/src/Blog/Handler/ListPostsHandler.php#L107-L120

https://github.com/laminas/getlaminas.org/blob/3c490f76e05ec9f2e9653f1d16ac117defa376aa/src/Blog/ConfigProvider.php#L80-L83

File: src/Blog/templates/post.phtml [hunk 2]
```diff
@@ -20,23 +20,23 @@ $this->end();
...
             <div class="e-content">
             <?= $post->body ?>
 
             <?= $post->extended ?>
             </div>
         </article>
+
+        <aside class="col-md-4 d-none d-sm-block sidebar blog">
+            <p>
+                Written on <time class="dt-published" datetime="<?= $this->formatDateRfc($post->created) ?>"><?= $this->formatDate($post->created) ?></time>.
```
Comment (michalbundyra) [inline src/Blog/templates/post.phtml]: Should we update it to "Written by ... on ... " ?

Comment (weierophinney) [reply src/Blog/templates/post.phtml]: Either one works for me.

Comment (michalbundyra) [reply src/Blog/templates/post.phtml]: @weierophinney The problem is that there is no any information about the author with the blog post.

**Pull request merged**